js/file.js

```diff
@@ -0,0 +1,18 @@
+function showCloseMenu() {
+  const menuSection = document.querySelector('#mobile-nav');
+  const nav = document.querySelector('#desktop-nav');
+  const headerName = document.querySelector('#logo');
+  const itemsContainer = document.querySelector('#items-container-menu');
+
+  menuSection.classList.toggle('d-none');
+  nav.classList.toggle('d-none');
+  headerName.classList.toggle('d-none');
+  headerName.classList.toggle('d-flex-center');
+  itemsContainer.classList.toggle('d-none');
+  itemsContainer.classList.toggle('d-flex');
```

Comment:
- [ ] [OPTIONAL] As you can see we need a few steps to make sure we are following the design, could you please implement them?

1- [OPTIONAL] `Portfolio`, `About`, and `Contact` should expand the entire width
2- `X` button should be on the right side